### DISCUSSION TITLE
'Invalid "--config" value: Cannot find "serverless-blaash-microservices.tmp.yml" in service folder

### DISCUSSION THREAD

**Discussion by ghost** [serverless/serverless#12032] [Q&A]
D:\dev\microservices-node>npx serverless mono --nestApp blaash-microservices --command offline
serverless-nest-monorepo: Serverless NestJS Monorepo Plugin Initialized for: blaash-microservices
serverless-nest-monorepo: Building Nest app: blaash-microservices...
serverless-nest-monorepo: Nest app blaash-microservices build finished.
serverless-nest-monorepo: Using serverless configuration from apps/blaash-microservices/serverless.yml
serverless-nest-monorepo: Running: serverless offline --config serverless-blaash-microservices.tmp.yml
Environment: win32, node 18.16.0, framework 3.32.2 (local), plugin 6.2.3, SDK 4.3.2
Docs:        docs.serverless.com
Support:     forum.serverless.com
Bugs:        github.com/serverless/serverless/issues

Error:
'Invalid "--config" value: Cannot find "serverless-blaash-microservices.tmp.yml" in service folder

Exit code: 1
serverless-nest-monorepo: Cleaning up: serverless-blaash-microservices.tmp.yml

D:\dev\microservices-node> 
I am facing this issues while deploying serveless mono repo nestjs app on aws.
Can any please suggest me how to resolve it?

**Comment by alvincrisuy** [comment]
Hi, what is your solution? have you resolved your issue?
